gradle/publish-runnable.gradle

```diff
@@ -34,7 +34,7 @@ bintray {
     publish = true
     pkg {
         repo = 'releases'
-        name = 'groovy-language-server'
+        name = project.ext.get("applicationName")
```

Comment:
+1, let's just use `project.name` here

If you want to use `ext` in the future (to specify additional properties), in this file you would need:
```
ext {
    applicationName
}
```

and then in the subproject build.gradle, add:
```
ext.applicationName = 'myname'
```